common/wforce-webserver.cc

```diff
@@ -203,7 +204,15 @@ void WforceWebserver::connectionThread(WforceWebserver* wws)
 
       req.getvars.erase("_"); // jQuery cache buster
 
+      // Generate HTTP Date format string
+      char datebuf[WFORCE_MAX_DATE_STRING_LEN];
+      time_t now = time(0);
+      struct tm tm = *gmtime(&now);
+      std::strftime(datebuf, sizeof datebuf, "%a, %d %b %Y %H:%M:%S GMT", &tm);
```

Comment:
```suggestion
      std::strftime(datebuf, sizeof(datebuf) - 1, "%a, %d %b %Y %H:%M:%S GMT", &tm);
```

Reply:
Well the strftime man page says that the \0 is taken into account:
````
 No more than maxsize characters will be placed into the array.  If the total number of resulting charac-
     ters, including the terminating NUL character, is not more than maxsize, strftime() returns the number
     of characters in the array, not counting the terminating NUL.  Otherwise, zero is returned and the
     buffer contents are indeterminate.
````

Reply:
However that does also make me think that I should at least zero out the buffer if strftime returns 0.

Reply:
> Well the strftime man page says that the \0 is taken into account:

Indeed. Sorry